corehq/apps/hqcase/views.py

```diff
@@ -81,7 +82,8 @@ def delete_cases(self, request, domain):
 @csrf_exempt
 @allow_cors(['OPTIONS', 'GET', 'POST', 'PUT'])
 @api_auth
-@require_can_edit_data
+@require_permission(Permissions.edit_data)
```

Comment:
are `@require_can_edit_data` and `@require_permission(Permissions.edit_data)` essentially equivalent?

Reply:
yep:
https://github.com/dimagi/commcare-hq/blob/cb31922c7f001b5f2abe89dc4f8204f41adb8ad9/corehq/apps/case_importer/views.py#L45-L45
Just switched it to maintain parity between the two